presto/utils.py

```diff
@@ -33,6 +34,9 @@
 device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
 DEFAULT_SEED: int = 42
 
+NODATAVALUE: int = 65535
+MIN_EDGE_BUFFER: int = 2  # Min amount of timesteps to include before/after the valid position
```

Comment:
we either have to make this a more informative variable name or move it to where we need it (only in dataset.py?) Is this only used in case of augmentation? Then we could also add this to the name. TBD.

Reply:
moved these two to dataops.py, since they are used both in dataset.py and utils.py. looks like it is the best place for now, since circular imports are avoided in this case, and semantically dataops.py contain similar parameters e338c0915f78e59bfa6074f64627b9d933b0752d